### PR TITLE
Update icons embeding

### LINKED ISSUE #391: SVG icons embedding cross-origin issue
screen shot 2015-10-01 at 13 02 31

### PR DESCRIPTION
**Pull request by aju**
closes #391

### CODE REVIEW

File: src/svg-polyfill.js [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
-    svg.appendChild(symbols[i]);
-  }
-
-  parentBody.appendChild(svg);
-}
+(function() {
+  document.addEventListener('DOMContentLoaded', function() {
+    var svgWrapper = document.createElement('div');
+    svgWrapper.style.display = 'none';
+    svgWrapper.innerHTML = '#SVG#';
```
Comment (kdzwinel) [inline src/svg-polyfill.js]: `#SVG#` is a placeholder where icons will be injected by gulp

File: scripts/tasks/svgs-generate.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ function svgSymbolCleanUp(shape, sprite, callback) {
 }
 
 function svgAddPolyfill(svgPolyfill, svg) {
-    return svg.replace('</svg>', '<script xmlns="http://www.w3.org/2000/svg" type="text/ecmascript">' + svgPolyfill + '</script></svg>');
+    return svgPolyfill.replace('#SVG#', svg.replace(/(\r\n|\n|\r)/gm,""));
```
Comment (kdzwinel) [inline scripts/tasks/svgs-generate.js]: Removes line breaks and puts SVG content into JS (replacing `#SVG#` placeholder)

**Comment by kdzwinel** [comment]
OK

File: src/docs/_layouts/default.html [hunk 3]
```diff
@@ -13,11 +13,12 @@
     <link rel="stylesheet" href="../style-guide.css">
     <link rel="stylesheet" href="{{ "/css/main.css" | prepend: site.baseurl }}">
 
+    <script src="images/icons.js"></script>
```
Comment (vergilius) [inline src/docs/_layouts/default.html]: its worth noticing that this doesn't need to be included in head

**Comment by vergilius** [comment]
looks :+1:

**Pull request merged**